docs/ref/contrib/sitemaps.txt

```diff
@@ -362,8 +362,8 @@ Here's what the relevant URLconf lines would look like for the example above::
     from django.contrib.sitemaps import views
 
     urlpatterns = [
-        url(r'^sitemap\.xml$', views.index, {'sitemaps': sitemaps}),
-        url(r'^sitemap-(?P<section>.+)\.xml$', views.sitemap, {'sitemaps': sitemaps},
+        path('sitemap.xml', views.index, {'sitemaps': sitemaps}),
+        path('sitemap-<section>.xml', views.sitemap, {'sitemaps': sitemaps},
```

Comment:
The old version allowed anything, including slashes (not sure if ententional). Should this use `<path:section>` instead?

Reply:
No I don't think so. Looks like it *ought* to be a `str` converter here.